test/basic.js

```diff
@@ -198,6 +198,27 @@ test('add 3 messages', (t) => {
   )
 })
 
+test('add three messages in batch', (t) => {
+  const rando = ssbKeys.generate()
+  const post4 = { type: 'post', text: 'd' }
+  const post5 = { type: 'post', text: 'e' }
+  const post6 = { type: 'post', text: 'f' }
+
+  let s = validate.initial()
+
+  s = validate.appendNew(s, null, rando, post4, Date.now() - 3)
+  s = validate.appendNew(s, null, rando, post5, Date.now() - 2)
+  s = validate.appendNew(s, null, rando, post6, Date.now() - 1)
+
+  const msgVals = s.queue.map((kvt) => kvt.value)
+  db.addBatch(msgVals, (err, keys) => {
+    t.error(err, 'no err')
+    t.equals(keys.length, 3)
+    t.deepEquals(keys, [undefined, undefined, undefined])
```

Comment:
why are these undefined? I'm assuming we are doing the `pluck: 2` to get the kvts?

And it seems `addOOOBatch` would want `pluck: 2` as well?

Reply:
Oh, well, I made addBatch return undefined on purpose because I thought that we didn't need any output. So I used `pluck: 2` to create those undefined (reminder: pluck 2 means we pick the **third** argument in the callback, which is undefined, the **first** argument would be the `err` and the **second** argument, i.e., pluck 1, would be `res`).

I can change these, I just want to know where will these keys be used as an output.

Reply:
I think that would be best to keep it consistant with `add`